package-utils.js

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
+const path = require('path');
+const fs = require('fs');
+
+const modifyPackageJson = (packageModifier) => {
+  const packageJsonPath = path.resolve(process.cwd(), 'package.json');
+
+  const package = fs.readFileSync(packageJsonPath, { encoding: 'utf8' });
+  const parsed = JSON.parse(package);
+  const modified = packageModifier(parsed);
+  let stringified = JSON.stringify(modified, null, 2) + '\n';
+
+  //add new line at the end
+  stringified += '\n';
```

Comment:
Anstatt hier mit `let` zu arbeiten könntest du die 2. Leerzeile auch einfach in Zeile 11 hinten mit dran hängen?

Reply:
changed